code/modules/projectiles/projectile/beams.dm

```diff
@@ -62,6 +62,7 @@
 	damage = 15
 	irradiate = 300
 	range = 15
+	armour_penetration = 100
```

Comment:
How about buffing the damage, lowering armour penetration and making its armour flag check for radiation suit protection?